src/applications/vaos/utils/error.js

```diff
@@ -1,13 +1,25 @@
 import * as Sentry from '@sentry/browser';
+import { recordVaosError } from './events';
+
+export function captureError(err, skipRecordEvent) {
+  let eventErrorKey;
 
-export function captureError(err) {
   if (err instanceof Error) {
     Sentry.captureException(err);
+    eventErrorKey = err.message;
   } else {
     Sentry.withScope(scope => {
       scope.setExtra('error', err);
+      const message = `vaos_server_error${
+        err?.[0]?.title ? `: ${err?.[0]?.title}` : ''
+      }`;
+      eventErrorKey = message;
       // the apiRequest helper returns the errors array, instead of an exception
-      Sentry.captureMessage(`vaos_server_error: ${err?.[0]?.title}`);
+      Sentry.captureMessage(message);
     });
   }
+
+  if (!skipRecordEvent) {
+    recordVaosError(eventErrorKey);
```

Comment:
I don't think we need to fix it in the this PR, but we're not logging a bunch of stuff under the `vaos-error` event label and not all of those things are actually errors. The eligibility failures we record in `recordEligibilityGAEvents` are just failed checks, not errors.

Reply:
I agree that the eligibility failures are confusing.  What do you think about not categorizing those as `vaos-error`s?  Something like `eligibility-check-failed-${code-name}` or `ineligible-${code-name}`.  Any errors related to fetching would still count as a `vaos-error`.

CC: @nedierecel

Reply:
Please disregard.  I missed the conversion on the thread in https://github.com/department-of-veterans-affairs/va.gov-team/issues/5983#issuecomment-598762660.